### PR TITLE
add a flag for small letters to Shiritori

### PR DESCRIPTION
**Pull request by Riolku**
Similar to #213. Default true, since that's the current behaviour.

### CODE REVIEW

**Comment by Riolku** [comment]
I'm probably going to PR another flag for dakutened characters very soon, so if you want, you can wait to deploy until then :).

File: bot/src/discord_commands/shiritori_forever.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ module.exports = {
   hidden: false,
   uniqueId: 'shiritoriforever',
   requiredBotPermissions: [Permissions.embedLinks, Permissions.sendMessages],
+  requiredSettings: [
+    'shiritori_forever/lax_long_vowels',
+    'shiritori_forever/small_letters',
+  ],
```
[the remark this answers is not in the document]
Comment (Riolku) [reply bot/src/discord_commands/shiritori_forever.js:44]: Sure, one sec

**Comment by Riolku** [comment]
Sorry there's a garbage file in the commit one sec.

**Comment by Riolku** [comment]
nvm, I'm mistaken.

**Comment by mistval** [comment]
👍 Seems good. Since you said you may have another PR coming, I'll hold off on deploying for now.

**Pull request merged**